src/App.tsx

```diff
@@ -1,42 +1,89 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
 import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
+import { UserSelector } from './components/UserSelector/UserSelector';
 import { Loader } from './components/Loader';
+import { Post, User } from './types';
+import { getPosts, getUsers } from './api/dataFromServer';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+
+  const [loadingPosts, setLoadingPosts] = useState(false);
```

Comment:
boolean variable should start with `is,has`/ Fix in all cases
```suggestion
  const [isLoadingPost, setIsLoadingPosts] = useState(false);
```